### DISCUSSION TITLE
Distillation column solution is not converging

### DISCUSSION THREAD

**Discussion by vishakha-618** [DanWBR/dwsim#579] [General]
Hii daniel

I tried it so many times that my distillation is not getting converged, please check it. I took the sample as reference.
In the sample,  feed stage is not mentioned, even though it is running, how?
![Screenshot 2024-01-17 221552](https://github.com/DanWBR/dwsim/assets/156805573/9988af44-2a1e-4639-902f-abd529b264eb)
![Screenshot 2024-01-17 221305](https://github.com/DanWBR/dwsim/assets/156805573/beae801e-321c-4c6c-957a-6486f1435aac)

**Comment by DanWBR** [comment]
@vishakha-618 I will never be able to help you if you do not provide the flowsheet file!
